src/api/helpers/ApiRequestTypes.ts

```diff
@@ -0,0 +1,24 @@
+export interface RequestConfigType {
+  method?: string;
+  token?: string;
+  params?: unknown
```

Comment:
I think its bad idea using `unknown` type. We can make union type for this part. For example:
```
export type RequestParams = 
  | IRequestLogin
  | IRequestRegister
  | IRequestRecoveryPassword
  | IRequestRefreshToken
  | Record<string, unknown>;
  ```
  And use like this  `params?: RequestParams;`
  
  But better if we make types for each request personal. This is an example from my project (with axios, but also works with classic fetch):
  ```
  export const userInfo = (dataUserInfo: { isCheckedOutSubscriptionPlans: boolean }) => {
  return instance.patch<userInfoResponse>('/user-info', dataUserInfo, {
    headers: { Authorization: `Bearer ${Cookies.get('jwt')}` },
  });
};
  ```

Reply:
Fix this